cms/views.py

```diff
@@ -348,3 +353,45 @@ def change_password(request):
     context['form'] = form
     context.update(csrf(request))
     return render(request, 'cms/templates/change_password.html', context)
+    
+def confirm_student(request, mdlid):
```

Comment:
```
user_hashid = 
mdluserid = user_hashid.decode(mdlid)
```

can be replaced with 

```
mdluser_id = Hashids(salt=settings.SPOKEN_HASH_SALT).decode(token)[0]
```

and just cross if you need to handle any exception.